beater/eventsapibeat.go

```diff
@@ -97,6 +99,14 @@ func (e *EventsAPIBeat) Run(b *beat.Beat) error {
 
 	if e.config.SignInAttempts.Enabled {
 		e.log.Info("Starting sign-in attempts loop")
+		jwt, err := utils.ParseJWTClaims(e.config.SignInAttempts.AuthToken)
+		if err != nil {
+			return err
+		}
+
+		if !jwt.Features.Contains(utils.SignInAttemptsFeatureScope) {
+			return errors.New("sign-in attempt token does not have sign-in attempt feature")
```

Comment:
Should this be an error? It's possible for someone to choose not to include a feature scope in a token, and we should just not request that type of events, rather than resulting in an error.

Reply:
This is only checked if the user enabled SignInAttempts in the config and provided a token specifically for SignInAttempts so I think an error here is fine.

Reply:
Ah I see, that make sense then.